app/src/main/res/values/dimens.xml

```diff
@@ -8,6 +8,7 @@
     <dimen name="play_outline_size_big">150dp</dimen>
     <dimen name="timer_padding">24dp</dimen>
     <dimen name="tmb_shadow_height">50dp</dimen>
+    <dimen name="photo_side_slider_width">150dp</dimen>
     <dimen name="video_side_slider_width">150dp</dimen>
```

Comment:
Should I join these together and name "media_side_slider_width"?

Reply:
ye join them